pkg/controller/storagecluster/cephcluster.go

```diff
@@ -379,7 +387,7 @@ func newStorageClassDeviceSets(sc *ocsv1.StorageCluster, serverVersion *version.
 			// enable the new behavior while the console is waiting
 			// to be updated.
 			// TODO: Remove this behavior when OCP console is updated
-			count = count / 3
+			count = count / defaults.DeviceSetReplica
```

Comment:
This doesn't seem related to this PR, please remove it.

Reply:
Ok. I'll send it in a different PR.